numpy/core/src/umath/ufunc_object.c

```diff
@@ -2411,16 +2581,27 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
         if (i >= nin) {
             int dim_offset = ufunc->core_offsets[i];
             int num_dims = core_num_dims[i];
+            int has_flexible=0;
```

Comment:
spaces around `=`

Also, can we call this `num_flexible`, since that is what it ends up being?

Reply:
fixed